### DISCUSSION TITLE
What is the desired behaviour of `tightLists` option?

### DISCUSSION THREAD

**Discussion by half0wl** [aguingand/tiptap-markdown#52] [General]
The [API docs](https://github.com/aguingand/tiptap-markdown/tree/155d5fdd25299a3835feb5edfc601bb9ff68a13a#options) specify that setting `tightLists=true` will not render `<p>`s inside `<li>`s:

```
  tightLists: true,            // No <p> inside <li> in markdown output
```

However, I still see `<li><p></p></li>` in my output with `tightLists=true`:

![ss 2023-12-04 at 3 24 25 AM](https://github.com/aguingand/tiptap-markdown/assets/24421625/1af965ba-90f8-4234-86b2-3cee5df8a875)

And the test here seems to suggest that this is intended:
https://github.com/aguingand/tiptap-markdown/blob/155d5fdd25299a3835feb5edfc601bb9ff68a13a/__tests__/tight-lists.spec.js#L26-L31

So, what's the desired behaviour for this option -- as the docs specify, or as the code is doing?

**Comment by aguingand** [comment]
The core tiptap BulletList is set to always have a `<p>` inside each item so this package adds an additinal attribute to render a markdown tight list :
```markdown
- aaa <!-- <li>aaaa</li> -->
- bbb
```
If you set `tightList` to false it will render
```markdown
- aaa <!-- <li><p>aaa</p></li> -->

- bbb
```
